packages/cli/src/Queue.ts

```diff
@@ -44,26 +36,18 @@ export class Queue {
 	constructor(private activeExecutions: ActiveExecutions) {}
 
 	async init() {
-		const bullPrefix = config.getEnv('queue.bull.prefix');
-		const prefix = getRedisPrefix(bullPrefix);
-		const clusterNodes = getRedisClusterNodes();
-		const usesRedisCluster = clusterNodes.length > 0;
-
 		const { default: Bull } = await import('bull');
+		const { RedisClientService } = await import('@/services/redis/redis-client.service');
```

Comment:
Perhaps it should be `Queue` that is dynamically imported from `WorkflowRunner` instead.